app/forms/api/profile_creation_form.rb

```diff
@@ -57,6 +57,12 @@ def complete_session
         create_gpo_entry
       elsif phone_confirmed?
         if pending_in_person_enrollment?
+          applicant = user_session[:idv][:applicant]
+          UspsInPersonProofing::EnrollmentHelper.new.save_in_person_enrollment(
+            user,
+            profile,
+            applicant,
```

Comment:
Based on order of logic, could we be passing PII from what's assigned into `session[:pii]` above (in `create_profile`) ?

Reply:
(I think this change may also indirectly fix the failing spec)

Reply:
Fixed in 5b13b99a703bed46db1653ed27b636a8e451d081